lib/tictactoe/game.rb

```diff
@@ -15,41 +20,73 @@ def draw
       @board.draw
     end
 
-    def check_position(pos)
-      if @board.positions[pos] != "-"
-        raise "Error"
+    def get_symbol()
+      puts "Enter symbol"
+      while true
+        symbol = @stdin.gets.chomp
+        if !@validation.check_input_symbol(symbol)
+        else return symbol         end
```

Comment:
Can you turn on the format on save setting? This way you can format the code as you go.